dockstore-webservice/src/main/resources/openapi.yaml

```diff
@@ -7329,6 +7363,8 @@ components:
         - repository
         - sourceControl
         - workflow_path
+      discriminator:
+        propertyName: type
```

Comment:
Does this mean that Service and BioWorkflow has a property name called "type" or...?

Reply:
Yes, in order for Jackson to tell which subclass is intended (when sending an incomplete JSON that could be a service or a workflow) the type properties is needed. 
A bit more info on how this works in the annotations on top of Workflow